app/modules/purchase/refundable.rb

```diff
@@ -104,6 +120,11 @@ def refund_and_save!(refunding_user_id, amount_cents: nil, is_for_fraud: false)
         false
       rescue ChargeProcessorInvalidRequestError => e
         logger.error "Charge refund encountered an invalid request error in purchase: #{external_id}. Response: #{e.message}. #{e.backtrace_locations}"
+        if e.message&.include?("No such charge")
```

Comment:
I would not rely on something like this, as it's a bit risky. E.g. what if the Stripe account is in Spanish?